### PR TITLE
chore: update tested versions

### PR DESCRIPTION
**Pull request by kuisathaverat**
## What does this PR do?

Update versions tested with the APM Server master branch.

### CODE REVIEW

File: tests/versions/ruby.yml [hunk 1]
```diff
@@ -3,5 +3,5 @@ RUBY_AGENT:
   # release;latest -> gem elastic-apm
   # release;version -> gem elastic-apm, version
   - 'github;master'
-  - 'github;2.x'
```
Comment (kuisathaverat) [inline tests/versions/ruby.yml:6]: NIT: only changes the order of the versions to follow the same rules for all APM Agents.

File: tests/versions/python.yml [hunk 2]
```diff
@@ -4,5 +4,4 @@ PYTHON_AGENT:
   # release;release -> pip install elastic-apm==version
   - 'github;master'
   - 'release;latest'
-  - 'release;5.3.2'
```
Comment (kuisathaverat) [inline tests/versions/python.yml:7]: `Latest` should test the 5.x versions

File: tests/versions/java.yml [hunk 3]
```diff
@@ -1,6 +1,4 @@
 JAVA_AGENT:
   - 'github;master'
-  - 'release;1.0.1'
```
Comment (kuisathaverat) [inline tests/versions/java.yml:3]: really old versions, we add only the latest release

**Review by cachedout — APPROVED** [review]

File: tests/versions/python.yml (hunk 2 above)
Comment (basepi) [reply tests/versions/python.yml:7]: 6.x is upcoming so we'll need to remember to add the last 5.x version here at that time.

**Review by basepi — APPROVED** [review]

**Pull request merged**